src/Console/Command/FixCommand.php

```diff
@@ -506,6 +434,28 @@ protected function execute(InputInterface $input, OutputInterface $output)
     }
 
     /**
+     * @param string          $format
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return ReportInterface
+     */
+    protected function createReport($format, InputInterface $input, OutputInterface $output)
+    {
+        $reportClass = self::$reports[$format];
+
+        /** @var ReportInterface $report */
+        $report = new $reportClass();
+        $report->setDecoratedOutput($output->isDecorated());
```

Comment:
This is only for `text` report. it should not be part of ReportInterface.
Please remove it and simply split `text` report to `text` and `decorated-text` reporters.